freqtrade/strategy/interface.py

```diff
@@ -861,23 +862,22 @@ def min_roi_reached(self, trade: Trade, current_profit: float, current_time: dat
         else:
             return current_profit > roi
 
-    def ft_check_timed_out(self, side: str, trade: Trade, order: Dict,
+    def ft_check_timed_out(self, side: str, trade: LocalTrade, order: Order,
                            current_time: datetime) -> bool:
         """
         FT Internal method.
         Check if timeout is active, and if the order is still open and timed out
         """
         timeout = self.config.get('unfilledtimeout', {}).get(side)
-        ordertime = arrow.get(order['datetime']).datetime
         if timeout is not None:
```

Comment:
This is weird. Function applies static timeout if `unfilledtimeout`, but it is a required property. I think it should align with other callbacks and not be used if custom callback is used.

Reply:
I'm not sure i'd change the behavior of unfilled-timeout at the moment (i don't really see a reason to)
the documentation is quite clear on this:

> For the above example, unfilledtimeout must be set to something bigger than 24h, otherwise that type of timeout will apply first.

if you think about it - it also simplifies things to some extend - if i wanna cancel the order if price moved the wrong direction - or after 5 minutes - i can leave "5" as timeout - and simply care about the "price moved" segment.